README.md

```diff
@@ -17,7 +17,29 @@
 
 ## Usage
 
-> Some usage instructions
+####scripts
+
+to: | enter
+:---:|---:
+start nodemon* |`npm run start `
+start webpack**|`npm run build`
+seed the DB | `npm run seed`
+run tests| `npm run test`
+check coverage| `npm run test:coverage`
+
+
+*will watch _only_ the server, _not_ the DB.
+**in watch mode.
+
+####API
+
+|REST|API ENDPOINTS|
```

Comment:
Could be helpful to have reference to the fields for creating or updating through the API.